### PR TITLE
chore: update 2d space shooter to ngo 1.0.0 [MTT-3424]

### PR DESCRIPTION
**Pull request by LPLafontaineB**
Simple bump to latest NGO version which has some API changes.

Ticket: [MTT-3424](https://jira.unity3d.com/browse/MTT-3424)

### CODE REVIEW

**Comment by LPLafontaineB** [comment]
Will mark as ready for review once pre.10 is released

**Review by fernando-cortez — APPROVED** [review]

File: Basic/2DSpaceShooter/Assets/Scripts/RandomPositionPlayerSpawner.cs [hunk 1]
```diff
@@ -43,9 +44,20 @@ private void Awake()
         networkManager.ConnectionApprovalCallback += ConnectionApprovalWithRandomSpawnPos;
     }
 
-    void ConnectionApprovalWithRandomSpawnPos(byte[] payload, ulong clientId, NetworkManager.ConnectionApprovedDelegate callback)
+    void ConnectionApprovalWithRandomSpawnPos(NetworkManager.ConnectionApprovalRequest request, NetworkManager.ConnectionApprovalResponse response)
```
Comment (SamuelBellomo) [inline Basic/2DSpaceShooter/Assets/Scripts/RandomPositionPlayerSpawner.cs:47]: Do we need connection approval at all at this point? There's an option in the NetworkManager to disable it. Or do we have doc pointing to this? Why do we have a pending approval that doesn't do much? Could we point users to boss room for pending approval examples?

Comment (LPLafontaineB) [reply Basic/2DSpaceShooter/Assets/Scripts/RandomPositionPlayerSpawner.cs:47]: The connection approval here is responsible for determining the player spawning positions. But the pending approval is just a test that I forgot to remove. I'll fix this

Comment (SamuelBellomo) [reply Basic/2DSpaceShooter/Assets/Scripts/RandomPositionPlayerSpawner.cs:47]: player position can be set around player object spawn also, no need for connection approval for this.

Comment (SamuelBellomo) [reply Basic/2DSpaceShooter/Assets/Scripts/RandomPositionPlayerSpawner.cs:47]: But I guess we already have an example of this in ClientDriven. Ok, if you could add some comments on this for position, that'd be great.

**Review by SamuelBellomo — APPROVED** [review]

**Pull request merged**